### ISSUE TITLE
TWE: IS21: RP001: Intern Exit Interview Recordings: De-identify: Intern 003

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#478]
### Dependencies  
~~- [ ] All the user interview issues have been created~~ IS21 didn't set up those issues, but all interviews were conducted. See [TWE: IS21: RP001: Intern Exit Interview Roll Call and Session Table](https://docs.google.com/spreadsheets/d/1dTlIOhtZ5fCNo0GrfKHlOTbdvsRlPUaLIfHk8Cc7Zrw/edit?usp=sharing)
~~- [ ] Interview Recording Upload Form needs to be created~~ The interviews were conducted before we set up the upload form, so it doesn't apply in RP001's case
- [x] Interview recordings have been uploaded and moved into Internship - PII's "My Drive"
- [x] Interview transcripts have been created

### Overview 
We need to clean up and de-identify RP001's Intern Exit interview transcripts so that we can move on to the data analysis and insights generation phase. 

During these tasks, you will be asked to add links to the Resources/Instructions section, and that section is at the bottom of this issue (go view it now).

Another tip is to use split browser windows side by side when working on any GitHub issue, so you can edit the issue on one window, and check the resources or click through links on the other window to avoid a lot of back and forth. Make sure to refresh both windows to reflect any changes you made, including updating the issue or checking the boxes on one of them.

### Action Items
- [x]  UX lead adds the assignee to the Internship - PII's "My Drive" as a Contributor so that they have access to Internship - PII's My Drive where the recordings and transcripts with PII are stored.
   - [x] UX lead accesses the Internship - PII's "My Drive"
   - [x] Choose "Manage members", which is located towards the top right side of the browser <img width="200" alt="Screenshot 2023-08-09 at 6 09 36 PM" src="https://github.com/hackforla/Internship/assets/57029070/f7cc7fbe-1b00-4ded-8b44-b10fdbccae46">
   - [x] Enter the assignee's email address and select the role as "Contributor" <img width="300" alt="Screenshot 2023-08-09 at 6 11 40 PM" src="https://github.com/hackforla/Internship/assets/57029070/41ab3531-75b1-435a-836a-1b24be3ecb69">
   - [x] Confirm with the assignee that they have access to the Internship - PII's "My Drive" 

#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [x] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [x] Find relevant wiki page for RP001
    - [x] Copy the link for the wiki page
    - [x] In Resources # 2.01, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">

- Customize Resource for this Research Plan's Google Drive Folder
    - [x] Open the Google Drive's Research by Type Folder (Resources # 1.02)
    - [x] Choose the `Intern` folder
    - [x]  Choose the `RP001` folder
    - [x] Copy the link of the `RP001` folder
        1. Choose the three vertical dots on the right side of the `RP001` folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.02, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recordings and Transcriptions Tracking Sheet
    - [x] Open the Research Plan's Google Drive folder in the Internship's shared drive (Resource # 2.02)
    - [x] Locate the `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet` in the folder
    - [x] Copy the link of `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet`
        1. Choose the three vertical dots on the top right side of the file  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.03, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

- Customize Resource for Interview Recording Folder stored in the Internship - PII's `My Drive`
    - [x] Log into your Google account so you will be able to access the Internship - PII's 'My Drive' in the next steps
    - [x] Choose Shared drives > Internship - PII > Internship -PII
    - [x] Choose `TWE Interview Recordings by Plan #` folder in Internship - PII's `My drive` (Resources # 1.03) 
    - [x] Locate the video recording folder for RP001 inside the folder `TWE Interview Recordings by Plan #`
    - [x] Copy the link of the video recording folder for RP001 
        1. Choose the three vertical dots on the right side of the folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.04, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
 
- Customize Resource for the De-identified Transcripts Folder in the Shared Drive 
    - [x] Go to RP001 folder (Resources # 2.02)
    - [x] Locate the `RP001 De-identified Transcripts` folder
    - [x] Copy the link of the folder
        1. Choose the three vertical dots on the right side of the folder <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

- Customize Resources for the Recording and Transcript you are Assigned to
    - [x] Check the title of this issue to identify the participant number, which comes after `De-identify:`
    - [x] Open the `TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet` (Resources # 2.03)
    - [x] Locate the recording video in .mp4 format that matches the participant number in Column B of the tracking sheet
    - [x] Copy the link of the recording that matches the participant number in Column B of the tracking sheet
    - [x] In Resources # 2.06, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Locate the transcript in .txt format that matches the participant number in Column C of the tracking sheet
    - [x] Copy the link of the .txt file that matches the participant number in Column C of the tracking sheet
    - [x] In Resources # 2.07, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Prepare for cleaning up and de-identification by converting the .txt file into a Google Doc
        1. Click on the link in Column C `Transcription Link` that matches the participant number (Resources # 2.07)
        1. Choose `Open with Google Docs` [![Screenshot 2023-01-06 at 11 04 38 AM](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)](https://user-images.githubusercontent.com/57029070/211081813-b49a22f4-1de2-4a5f-a10a-e7c7686ae1d0.png)
        1. A Google Doc is generated in a new window with the same .txt file name 
        1. The Google Doc is now saved into the same folder with the corresponding video and .txt file
        1. Open the RP001 Interview Recording folder (Resources # 2.04)
        1. Open the corresponding folder for Intern 003 and check to make sure a Google Doc transcript has been created
   - [x] Make a copy of the Google Doc that was just generated and rename it
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Make a copy"
        1. A copy of the Google Doc is generated in the same folder       
        1. Copy 
        ```
        To be de-identified-
        ``` 
        1. Go back to the folder where a copy of the Google doc was created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Rename"
        1. Remove `Copy of` from the file name when the Rename text box pops up
        1. Paste what you copied into the beginning of the file name text box
        1.  Choose "Ok" 
    - [x] Copy the link of the "to be de-identified" Google Doc transcript you just created
        1. Choose the three vertical dots on the top right side of the file <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] In Resources # 2.08, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Clean up and de-identify the transcript
- [x] Open the Google Doc transcript if it is not open yet (Resource # 2.08)
- [x] Listen to the recording to get yourself familiar with the transcript
- [x] Go to Resources # 1.04 to get a refresher on the cleaning up and de-identification process. **_This is particularly important if it is your first time cleaning up an interview transcript_**. 
    - [x] Check Resources # 1.04.01 to understand when and how to use brackets `[ ]` and parenthesis `( )` when working on cleaning up the transcript, so you can learn to use them properly.
    - [x] Watch the videos in Resources # 1.04.02 as they walk you through how to clean up and de-identify the transcript
    - [x] Read Resources# 1.04.03 and 1.04.04 for more best practices to clean up and de-identify an interview transcript
    - [x] Follow the following guidelines on the transcript formatting:
        - [x] Use the 'in-line' format as seen [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit)
        - [x] Use 6 digit format for all the timestamps, for example, 01:01:01 as hour one/minute one/second one
        - [x] Use "Interviewer" and "Intern ###" throughout the transcript. See an example [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit).
            - If there were more than two people recorded on the transcript, name the third person based on the interview set-up or context, such as Program Manager, Note Taker, etc. If you are not sure what to name them, ask leads on the project to clarify.
        - [x] If there are other numerical values in the transcript, transcribe them using texts instead of numbers so we can easily scan the transcript using timestamps as they are in numbers. For example, if a time such as '6:30' is mentioned, type out 'six thirty' when cleaning up the transcript. Another example is if the interviewee mentioned he mainly sought help from '3 other peers', type out 'three' instead of using the number '3'.
- [x] Listen to the interview recording again and clean up the inaccuracies in the transcript as you read along
    - [x] Separate the texts based on speakers and timestamps (timestamps should match the video file timestamps in case a researcher needs to go back and double-check the original video)
    - [x] Add any missed words if the transcribing process missed any or to provide more context
    - [x] Write exactly what they say, even if it is grammatically incorrect or doesn’t make much sense. You can use brackets to correct them if needed, but no need to erase them.
    - [x] Edit any typos and misidentified words in the transcript because the transcribing software is not 100% accurate
    - [x] Deleted repeated words
    - [x] Keep interjections like “hmm”, “uh”, “woah”, “yeah”, “okay”, “um”, “ohh”, “mmm” because they often contain emotions, reactions, and meanings (e.g. Mmmm [no], Mhmm [yes])
    - [x] For verbal filler words such as "um" and "uh", keep a balance between authenticity and readability. See Resources # 1.04.04 for more on this topic.
        - When the verbal fillers are distractions and don't serve any purpose, you may remove them.
        - When the verbal fillers can indicate the interviewee's emotions or thoughts. In this case, keep the verbal fillers and provide context to demonstrate the emotions or thoughts. E.g., (The participant hesitated for a while before coming up with the answer).
    - [x] Where needed, add context in brackets (for important information) or parentheses (for supplemental information) so that a reader can understand what was happening without watching the video. E.g., he [the mentor] was helpful; or (steps off camera).
- Read through the transcripts again and make edits to de-identify any personal information
    - [x] Search for the participant's name 
    - [x] Replace with their participant number, i.e. Intern 001
    - [x] Search for unknown speakers in the transcript
    - [x] Replace unknown speaker with UXR number or other participant type/number
    - [x] Read through the transcripts to search for any identifiable information in the interview
    - [x] Replace any personally identifiable information with non-identifiable terms. For example, if someone's name is mentioned, we should use their roles inside square brackets so the reader knows this information was redacted (e.g., [team member], [peer], [mentor], [manager]). If their specific school is mentioned, we should redact that info with either [high school] or [college].
    - [x] If they mentioned any specific issues they worked on, make sure to remove the issue numbers and rephrase the issues they worked on
- Read through the edited transcripts again
    - [x] Focus on punctuation, readability, and formatting.
    - [x] One recommendation is to install LanguageTool Chrome Extension (see Resources 1.05) to clean up the punctuation and verbal ticks in the transcript
- When you're satisfied that the transcript is completely deidentified
    - [x] Click on the version history and select the most current version
    - [x] Click the three dots <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/Internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96"> and select "Make a copy", and the "Copy version" window pops up
    - [x] In the "Name" text box, remove the "Copy of" text at the beginning and the timestamp at the end from the file name
    - [x] Copy 
    ```
    De-identified Interview Transcript-
   ```
    - [x] Paste the text you just copied in front of the file name in the "Name" text box
        - The new file name should be formatted like "De-identified Interview Transcript-RP001-UX Researcher Abbreviation###-Intern Abbreviation###"
        - An example: "De-identified Interview Transcript-RP006-U007-I001" 
    - [x] Save the renamed de-identified Google doc transcript into the `RP001 De-identified Transcripts` folder. This is to make sure that the version history does not include any identifiable information.
        1. Click the folder icon under "Folder"
        1. Choose "All locations"
        1. Choose Shared drives > Internship > Internships > Research > Research by Type > Intern > folder for RP001 > RP001 De-identified Transcripts
        1. Choose "Select"
        1. Select "Make a copy"
        1. The copy of the file is generated in a new browser
        1. Dismiss the notification that says the file is now in the shared drive
    - [x] Copy the link of the de-identified transcript in the shared drive
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [x] Paste what you copied into the matching participant's cell in Column D `Transcription Link - de-identified` in the tracking sheet (Resources # 2.03)
    - [x] In Resources # 2.09, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [x] Choose "Update comment" in GitHub and make sure all the checkboxes above have been checked

 
#### 
- [x] Review with UX Lead
- [ ] Product sign-off
- [ ] UXR Lead or PM removes the assignee from the Internship - PII's My Drive when closing the issue if the assignee no longer needs to access the PII drive.

#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/Internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03* [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1lvP_k4qGKD05QzbHBtIpEl4B3X7ylzoh?usp=sharing)
1.04 [Guidelines to Interviews Page 8](https://drive.google.com/open?id=19580_-BXfHPC9gTcAj35O2nzAtdNwTxp&disco=AAAA_U2KB_A)
1.04.01 [Brackets vs. Parentheses](https://www.strose.edu/wp-content/uploads/2015/10/Brackets-Handout-Cynthia2.pdf)
1.04.02 [Video folder](https://drive.google.com/drive/folders/1WPzYSrhWYluIqigjsHKpSYIhpQf7XcM5)
1.04.03 [Cleaning Up Zoom Transcriptions for Qualitative Research](https://crdpala.org/2022/10/05/cleaning-up-zoom-transcriptions-for-qualitative-research/)
1.04.04 [Determining Best Practice for Filler Words in Captions and Transcripts](https://blogs.vcu.edu/birdlab/2020/01/09/determining-best-practice-for-filler-words-in-captions-and-transcripts/)
1.05 [LanguageTool Chrome Extension](https://chrome.google.com/webstore/detail/grammar-checker-paraphras/oldceeleldhonbafppcapldpdifcinji)

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan: RP001](https://github.com/hackforla/internship/wiki/Research-Plan-1:-IS21-Interns)
2.02 [Google Drive Folder: RP001](https://drive.google.com/drive/folders/1uKUY_izpds3dHInNkiexZsU32ELF_Hze?usp=share_link)
2.03 [TWE: IS21: RP001: Intern Exit Interview Recordings and Transcriptions Tracking Sheet](https://docs.google.com/spreadsheets/d/1Uw_KFUQ3n8DJrxwbc3vnMD8stgF6kTIwSUn5yP9xFuc/edit?usp=drive_link)
2.04* [RP001 Interview Recording Folder](https://drive.google.com/drive/folders/1R8sXXGrx47PNj2GpMSUIpMXYllAEgy19?usp=drive_link)
2.05 [RP001 De-identified Transcripts Folder](https://drive.google.com/drive/folders/1A2crr41KWwnoCvdtwmwJnmoP0IzH1Z54?usp=drive_link)
2.06 [Intern 003 Recording Link](https://drive.google.com/file/d/1eGp05HSvmnIcfFyATht1Q_2BoGlhI1QH/view)
2.07 [Intern 003 Transcript .txt File](https://drive.google.com/file/d/1HF60Z70eK_-i7aPVH1PwEfz8V5jFzENn/view)
2.08 [Intern 003 To be De-identified Transcript Google Doc](https://docs.google.com/document/d/1IyChTihvPCHqtLvxmC09DREpZAg1GQ4-FkApU0fCcZc/edit?usp=sharing)
2.09 [Intern 003 De-identified Transcript Google Doc](https://docs.google.com/document/d/1Yw6ZgiwPErnx8UfMv6r5yaBIRRAiC0duOF4F-CH1uyo/edit?usp=sharing)

*This folder can only be accessed from the Internship - PII's "My Drive"

**Comment by pandanista** [comment]
Hi @Dzotsen, I've assigned you to de-identify the transcript for intern 003 from cohort 2021. Please follow the action items and let me you know if you have any questions.

**Comment by pandanista** [comment]
Hi @Dzotsen, for the de-identification and cleaning-up process, those videos in the [video folder](https://drive.google.com/drive/folders/1WPzYSrhWYluIqigjsHKpSYIhpQf7XcM5) might be helpful. Please leave any feedback or thoughts for the videos in this issue so Drew can incorporate them into the videos he is producing.

**Comment by pandanista** [comment]
@Dzotsen Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.
Thank you.

**Comment by pandanista** [comment]
@Dzotsen Please provide update:

Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."
remember to add links to the top of the issue if they are going to be needed again.
Thank you.

**Comment by pandanista** [comment]
@Dzotsen Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@Dzotsen Welcome back, and hope your 2024 is off to a good start.

Would you please provide an update?
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@Dzotsen I just clarified with the leads that Adam was the tech lead for the website team during that time, not a mentor.

**Comment by pandanista** [comment]
Added a few guidelines on the transcript formatting. You are already following most of them. I am listing them below as a reference.

1. Use the 'in-line' format as seen [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit)
1. Use 6 digit format for all the timestamps, for example, 01:01:01 as hour one/minute one/second one
1. Use "Interviewer" and "Intern ###" throughout the transcript. See an example [here in Intern 001's transcript](https://docs.google.com/document/d/1C8-qEAot7Sw5rhpfuplP0D0SLxypdChUigC1u7LCWJA/edit).
    - If there were more than two people recorded on the transcript, name the third person based on the interview set-up or context, such as Project Manager, Note Taker, etc. If you are not sure what to name them, ask leads on the project to clarify.
1. If there are other numerical values in the transcript, transcribe them using texts. For example, if a time such as 6:30 is mentioned, type out six thirty when cleaning up the transcript. Another example is if the interviewee mentioned he mainly sought help from 3 other peers, type out three instead of using the number 3.

**Comment by pandanista** [comment]
@Dzotsen Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@Dzotsen Please provide update:
1. Progress: "What is the current status of your project? What have you completed and what is left to do?" - we know you just came back from break, please just summarize the progress from before then
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by Dzotsen** [comment]
I could not follow the instruction for this particular sequence. Instead, I downloaded the transcript and uploaded it to the corresponding folder. 
![Github](https://github.com/hackforla/internship/assets/124115467/c5a480ce-c072-40b5-ba03-c61d892bffbe)

**Comment by pandanista** [comment]
At the UXR team meeting on 2/13, we have figured out the reason why the instrcutions above were hard to follow. It was due to the version history was a unfamiliar feature for Google doc users, and the three dots were hard to find. Therefore, the UX lead will make changes to the template in #509.

**Comment by pandanista** [comment]
I've reviewed the transcript and made some minor edits.

@sunannie27 Would you please review the [Intern 003 De-identified Transcript Google Doc](https://docs.google.com/document/d/1Yw6ZgiwPErnx8UfMv6r5yaBIRRAiC0duOF4F-CH1uyo/edit?usp=sharing)?

**Comment by ExperimentsInHonesty** [comment]
@pandanista This transcript seems substantially different from others we have seen.  For instance, 
- there are brackets around the word like, in some places, and not in others.  Generally brackets are meant to denote words the person left out, but are needed to make the sentence make sense.
- other brackets in the transcript mention background noises and words that are unclear.  

When you were reviewing the transcript did you listen to the recording?

**Comment by pandanista** [comment]
I have reviewed the recording again, and cleared out the unclear wording in the transcript.

Also, the research team discussed the use of brackets at the UXR meeting on 2/27 and we decided to remove the brackets around the verbal fillers, such as like, but keep the verbal fillers so the transcript stays true to the communication style of the intern. We all agreed that verbal fillers wouldn't affect the outcome and validity of the research, so we were okay keeping them.
<img width="417" alt="Screenshot 2024-02-27 at 5 32 36 PM" src="https://github.com/hackforla/internship/assets/57029070/cf0289ab-6583-4c9a-8962-d633a007600c">


Next steps: @Dzotsen will remove the brackets around the verbal fillers and do another round of cleaning-up of the transcript.

**Comment by pandanista** [comment]
Thank you for being so patient with us, @Dzotsen.

Would you please update "interviewer" with "UXR 003" in [intern 003's transcript](https://docs.google.com/document/d/1Yw6ZgiwPErnx8UfMv6r5yaBIRRAiC0duOF4F-CH1uyo/edit?usp=sharing)? Much appreciated!

We decided to include the UXR numbers in the transcripts because 1). there might be different probing questions for semi-structured interviews from different interviewers, and 2). it will be easier to reference the UXR when doing analysis or making highlight reels (if needed).

**Comment by Dzotsen** [comment]
Intern 003 transcript updated.

**Comment by ExperimentsInHonesty** [comment]
@pandanista please define next steps for this issue.

**Comment by sunannie27** [comment]
@pandanista Looks good to me.

**Comment by ExperimentsInHonesty** [comment]
@Dzotsen Thanks for the great work on this.
